pandas/tests/series/methods/test_case_when.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+import pytest
+
+from pandas import (
+    DataFrame,
+    Series,
+    array as pd_array,
+    date_range,
+)
+import pandas._testing as tm
+
+
+@pytest.fixture
+def df():
+    """
+    base dataframe for testing
+    """
+    return DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
```

Comment:
Could you in-line this dataframe where used?

Reply:
it is used in a number of places; inlining it feels repetitive. is there a reason for this?